packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -215,4 +215,35 @@ void main() {
     expect(iconColor(expandedIconKey), _unselectedWidgetColor);
     expect(iconColor(collapsedIconKey), _accentColor);
   });
+
+  testWidgets('ExpansionTile subtitle', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      MaterialApp(
```

Comment:
It there a need for all the extra config? I would imagine that the following would suffice: 
```
      MaterialApp(
        home: Scaffold(
          body: ExpansionTile(
            title: const Text('Title'),
            subtitle: const Text('Subtitle'),
            children: const <Widget>[ListTile(title: Text('0'))],
          ),
        ),
      ),
```

This also makes your test's intent clear to future devs reading this.

Reply:
It isn't, initially I thought in doing this way, but I was unsure whether this additional config was wanted. I'm going to update it.